frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts

```diff
@@ -174,6 +174,7 @@ describe('Project Dashboard', () => {
     });
 
     it('is displayed when CR exists', async () => {
+      await browser.wait(until.not(until.presenceOf(projectDashboardView.launcherCard)));
```

Comment:
Were you able to reproduce? I'm not sure this is the issue. I suspect the `isPresent` call below happens asynchronously, and `createResource` completes too fast in some cases.

Reply:
ok, fixed to use `await`